Automattic/wp-calypso
# Site Design Picker : Remove button to use blank canvas

#### Changes proposed in this Pull Request

* Remove the button to use a blank theme from the site design picker

#### Testing instructions
* Checkout the branch
* Run yarn start
* Go to http://calypso.localhost:3000/
* Add a new site and step through onboarding 
* Click build to bring up design picker screen
* Check that the `Use a blank theme` button is not present


* Before
![image](https://user-images.githubusercontent.com/47489215/161097968-52e33ee4-41b5-4506-807a-4c107e972cf1.png)

* After
![image](https://user-images.githubusercontent.com/47489215/161098125-9529a9bf-5ccd-4df3-96b9-3228a494aa4d.png)




Related to #62384
